tests/io/test_readwrite.py

```diff
@@ -147,13 +147,14 @@ def test_incremental_io_on_disk(self, tmp_path: str, full_sdata: SpatialData):
         ]:
             sdata[name] = full_sdata[name]
             sdata.write_element(name)
+            sdata = read_zarr(sdata.path)
 
             with pytest.raises(
                 ValueError, match="The Zarr store already exists. Use `overwrite=True` to try overwriting the store."
             ):
                 sdata.write_element(name)
 
-            with pytest.raises(ValueError, match="Currently, overwriting existing elements is not supported."):
+            with pytest.raises(ValueError):  # TODO this gives different messages for different element types
```

Comment:
This is confusion, I will adjust it, but correct. Points, images and labels may be backed from disk. In that case the problem is that even with `overwrite=True`, we can't overwrite those elements, because overwriting them would mean to actually delete the source of the backed data.

The second error message occurs with non-backed images, labels, points or with shapes and tables (that currently are always non-backed). In that case we could overwrite the data but we don't do it because this could lead to data loss anyway (the case above is instead a "systematic data loss").

The only overwrite case that is supported is overwriting into a different Zarr store, for instance an old, non-loaded, spatialdata object.

Reply:
The workarounds I show in the tests work for both cases; I will document them in an example notebook and eventually we could automate their execution (after warning the user that this works only under particular assumptions).

Reply:
I will now make the error messages less cryptic.

Reply:
Done.